packages/sdk/src/clients/etherchain/client.ts

```diff
@@ -0,0 +1,26 @@
+import axios from "axios";
+import get from "lodash/get";
+
+type GasPrice = {
+  safeLow: number;
+  standard: number;
+  fast: number;
+  fastest: number;
+  currentBaseFee: number;
+  recommendedBaseFee: number;
+};
+
+export class Etherchain {
```

Comment:
Are we planning to use this in the bots? Shouldn't the bots and FE use the same logic if we want them to behave the same?

Reply:
> Are we planning to use this in the bots?
I don't know how to answer to this